course2_react/react-app2/src/Home.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import BlogList from "./BlogList";
+import { InfinitySpin } from "react-loader-spinner";
+import "./Home.css";
+import "./AddInput.css";
+import InputBox from "./InputBox";
+import useFetch from "./useFetch";
+
+const Home = () => {
+  const {
+    todos,
+    setTodos,
+    isPending,
+    setIsPending,
+    error,
+    setError,
+    newToDo,
+    setNewTODo,
+  } = useFetch("http://localhost:8001/todos");
+  //http://localhost:8000/todos
+
+  const deleteHandle = (id) => {
+    console.log(id);
+    const newToDo = todos.filter((todo) => todo.id !== id);
+    setTodos(newToDo);
+  };
+
+  const addTitle = () => {
+    let temp = { id: todos.length + 1, title: newToDo, completed: false };
```

Comment:
todos.length + 1 would not generate right Id always
eg. You have 3 items in Todo list with id's 1,2,3
Now you delete the second item ie. with id 2
So remaining list is 1,3

Now if you try to add a new entry the length of todolist is 2 so 2+1 is 3
But we already have 3 as a Id